walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,92 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.Comparator; 
+import java.util.List;
 
 public final class FindMeetingQuery {
-  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+  //public static final int START_OF_DAY = TimeRange.getTimeInMinutes(0, 0);
+  private static final int END_OF_DAY = TimeRange.getTimeInMinutes(23, 59);
+  //private static final int WHOLE_DAY = TimeRange.getTimeInMinutes(0, 24 * 60);
+  
+  private static final Comparator<Event> ORDER_BY_START = new Comparator<Event>() {
+    @Override
+    public int compare(Event a, Event b) {
+      return Long.compare(a.getWhen().start(), b.getWhen().start());
+    }};
+    
+  private List<TimeRange> acceptableMeetingTimes = new ArrayList<TimeRange>();
+  
+  public Collection<TimeRange> query(Collection<Event> eventCollection, MeetingRequest request) {
+    List<String> meetingAttendees = new ArrayList<String>(request.getAttendees());
+    List<Event> events = new ArrayList<Event>(eventCollection);
```

Comment:
Nit: Same, recommend scooting this down.

Reply:
Done